routes/routes.ts

```diff
@@ -243,10 +240,7 @@ router.delete(
             const accessToken = snapshot.val().accessToken;
             const newTwitterClient = new TwitterApi(accessToken);
             await newTwitterClient.v2.deleteBookmark(bookmarkedTweetId);
-            console.log("bookmark deleted successfully");
-            return res
-              .status(200)
-              .json({ message: "bookmark deleted successfully!" });
+            ResponseHandler.requestSuccessful(res)
```

Comment:
I think it would make sense for endpoints that alter data if the response message was more meaningful. The "request successful" message for an endpoint that deletes messages seems a bit ambiguous.